.github/workflows/jekyll-build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build Jekyll site
+
+on: [push, pull_request]
+
+permissions:
```

Comment:
do you need custom permissions for just the build part?

Reply:
I think we need some permissions to upload the artifact, no?

Reply:
Sure, but I think that's included in the default ones
id-token isn't default write and I assume you need that to do the actual deploy, but I don't think you need it to build